CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.0.58
```

Comment:
usually at the time of pull-request being submitted you dont know in what version it will be released, so rule of thumb is to keep it simply `## upcoming version`